src/pages/index.js

```diff
@@ -100,26 +100,28 @@ const IndexPage = () => (
         <ColSubheader>{'What charities are you recommending?'}</ColSubheader>
         <P>
           {
-            'We are not recommending any specific charities or organizations. Charitable donation is a personal choice, and we encourage people signing the pledge to make donations to organizations that matter to them personally. We intend to offer more resources on responsible giving (for example, how to find great non-profit organizations and evaluate their effectiveness), but the choice of recipient is up to the donor.'
+            'We are not recommending any specific charities or organizations. Charitable donation is a personal choice, and we encourage people signing the pledge to make donations to organizations that matter to them personally. We will offer signers resources on responsible giving (for example, how to find great non-profit organizations and evaluate their effectiveness), but the choice of recipient is up to the donor.'
           }
         </P>
 
         <ColSubheader>
           {
-            'Uber isn’t the only tech company with a potential IPO this year - should we be extending this to employees from Lyft, AirBnB, etc?'
+            'Uber isn’t the only company with a potential IPO this year. Can employees from other companies participate?'
```

Comment:
Maybe just simplify this to `Can employees from other companies participate?`?

Reply:
👍